src/agent/debuglet.ts

```diff
@@ -174,6 +183,9 @@ export class Debuglet extends EventEmitter {
 
     /** @private {Object.<string, Boolean>} */
     this.completedBreakpointMap_ = {};
+
+    /** @private {Array<number>} */
+    this.promiseResolvedTimestamp_ = Date.now();
```

Comment:
Can you set this value to either be `PROMISE_RESOLVE_CUT_OFF_IN_SECONDS` less than `Date.now()`, or some sentinel value? Otherwise Promises returned by `checkReady` when it's called within `PROMISE_RESOLVE_CUT_OFF_IN_SECONDS` within initialization will be incorrectly resolved.

Reply:
I think it's best to set this to `-Infinity` (or 0).

Reply:
Done.